azdo-task/DevcontainersCi/src/main.ts

```diff
@@ -23,10 +23,18 @@ export async function runMain(): Promise<void> {
 			);
 			return;
 		}
-		const devContainerCliInstalled = await devcontainer.isCliInstalled(exec);
+		const specifiedDevContainerCliVersion =
+			task.getInput('cliVersion') ?? 'latest';
```

Comment:
Same as below: Still use '0' as the default, so we can introduce breaking changes on the CLI by increasing its major version. Please use a constant, so we have it only once.

Reply:
See above resolution.